package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@blumilksoftware/eslint-config-blumilk",
```

Comment:
Why with @?

Reply:
I found some information about it here: [Configuring the destination repository](https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#configuring-the-destination-repository)